experiment_buddy/utils.py

```diff
@@ -38,6 +40,8 @@ def get_backend(ssh_session: fabric.connection.Connection, project_dir: str) ->
 def get_project_name(git_repo: git.Repo) -> str:
     git_repo_remotes = git_repo.remotes
     assert isinstance(git_repo_remotes, list)
+    if not len(git_repo_remotes):
```

Comment:
is this part of this PR?

Reply:
It helps with this PR, in the case the user skips the creation of the github repo, this tells exactly what's wrong when working with Buddy

Reply:
ok.